### PR TITLE
Add Manage Alerts to new Admin Portal

### PR DESCRIPTION
**Pull request by chathuranga95**
### Description
Adding Manage Alerts functionality to the revamped admin portal

##### Alert types and descriptions listing
alert tyles
===========================================================
email entering area

##### LeftMenu Icon
icon

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx [hunk 1]
```diff
@@ -229,6 +231,15 @@ const RouteMenuMapping = (intl) => [
             },
         ],
     },
+    {
+        id: intl.formatMessage({
+            id: 'Base.RouteMenuMapping.manage.alerts',
+            defaultMessage: 'Manage Alerts',
+        }),
+        path: '/analytics/manage-alerts',
+        component: () => <ManageAlerts />,
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx:240]: why we need to use lambda here ? component accept just component class/function, they have a separate prop for render function

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx [hunk 2]
```diff
@@ -0,0 +1,585 @@
...
+        marginBottom: theme.spacing(2),
+    },
+    alertConfigDialog: {
+        width: '60%',
+    },
+    configDialogHeading: {
+        fontWeight: '600',
+    },
+    btnContainer: {
+        marginTop: theme.spacing(),
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx:70]: Shall we use `[<Box mt={1}>](https://material-ui.com/components/box/#box)` to do this ?

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx [hunk 3]
```diff
@@ -0,0 +1,585 @@
...
+
+    useEffect(() => {
+        const supportedAlertsPromise = api.getSupportedAlertTypes();
+        const subscribedAlertsPromise = api.getSubscribedAlertTypesByUser();
+        Promise.all([supportedAlertsPromise, subscribedAlertsPromise]).then((response) => {
+            if (response[0].status === 204 || response[1].status === 204) {
+                setAnalyticsEnabled(false);
+            } else {
+                setAnalyticsEnabled(true);
+                setSubscribedAlerts(response[1].body.alerts);
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx:276]: Shall we first destructure these bodies and use them
i:e 
```javascript
[supportedAlerts, subscribedAlerts] = response;
```

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx (hunk 1 above)
Comment (chathuranga95) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/Base/RouteMenuMapping.jsx]: Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8662/commits/cb1fd45e6fbcb1c4166f90d86ef8f0c9366450f7

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx (hunk 2 above)
Comment (chathuranga95) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx]: Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8662/commits/ad2be95172c14f083670b0910b54a6435b9f97bc

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx (hunk 3 above)
Comment (chathuranga95) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ManageAlerts/ManageAlerts.jsx]: Fixed with commit: https://github.com/wso2/carbon-apimgt/pull/8662/commits/3dde61d37eba0356fd044756c4ccc61a48686a13

**Pull request merged**